plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridSequenceManager.java

```diff
@@ -64,7 +65,7 @@ protected GenericSequence createDatabaseObject(
     }
 
     @NotNull
-    public String buildStatement(@NotNull CubridSequence sequence, @NotNull boolean forUpdate) {
+    public String buildStatement(@NotNull CubridSequence sequence, @NotNull boolean forUpdate, @NotNull boolean hasComment) {

```

Comment:
Primitive values can't be null
```suggestion
    public String buildStatement(@NotNull CubridSequence sequence, @NotNull boolean forUpdate, boolean hasComment) {

```